src/Web/Offset/Splices.hs

```diff
@@ -63,14 +63,14 @@ wpCustomDateFill =
                                       (T.unpack wpFormat)
                                       (T.unpack date) :: Maybe UTCTime in
           case parsedDate of
-              Just d -> let dateSubs = subs [ ("wpYear", datePartFill "%0Y" d)
-                                            , ("wpMonth", datePartFill "%m" d)
-                                            , ("wpDay", datePartFill "%d" d)
-                                            , ("wpDate", datePartFill "%D" d) ] in
+              Just d -> let dateSubs = subs [ ("wpYear",     datePartFill "%0Y" d)
```

Comment:
@dhartunian Just noticed this! I pulled out a couple functions and it's much better now. :D